app/hotels/src/filter/__tests__/FiltersReducer.test.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+
+import FiltersReducer, { InitialFiltersReducerState } from '../FiltersReducer';
+
+const type = 'setFilter';
+
+describe('FiltersReducer', () => {
+  it('should return the initialState', () => {
+    // $FlowFixMe
```

Comment:
See `.flowconfig`:

> ; Please, do not use "fix me" suppress types and comments

If it's necessary (we are in tests so I am ok with that) use something else (`$FlowIssue` or `$FlowExpectedError` with an explanation).